### PR TITLE
Assess google report

### PR DESCRIPTION
**Pull request by michelbl**

### CODE REVIEW

File: _pages/en/google-transparency-report-assessment.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### The Google Cloud Big Query resource is unavailable
+
+Google [claims](https://support.google.com/transparencyreport/answer/9310667) that “The ads are published as a public data set on [Google Cloud BigQuery](https://console.cloud.google.com/marketplace/details/bigquery-public-data/google-political-ads). You can export a subset of the ads or access them programmatically.” However, we could not access the resource as of June 24th, 2019.
+
+### Data is updated irregularly
+
+Google claims that the data is updated weekly, which could be more frequent.
+
+We noticed that between June 14th 2019 and June 24th, the downloadable bundle has been updated twice: between June 14th noon and June 15th noon, and between June 18th noon and June 19th noon.
```
Comment (MattiSG) [inline _pages/en/google-transparency-report-assessment.md]: Could we get more data points on that update frequency?

File: _pages/en/google-transparency-report-assessment.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Limitations
+
+### The scope is limited to the European Parliament election
+
+As stated in the [Political advertising on Google FAQs](https://support.google.com/transparencyreport/answer/9310667), the Transparency Report “covers any ad that mentions a political party, candidate or current officeholder for the European Parliament”. Consequently, ads paid by political parties that did not campaign for the European Parliament elections do not (yet?) appear in the report.
+
+### The Google Cloud Big Query resource is unavailable
+
+Google [claims](https://support.google.com/transparencyreport/answer/9310667) that “The ads are published as a public data set on [Google Cloud BigQuery](https://console.cloud.google.com/marketplace/details/bigquery-public-data/google-political-ads). You can export a subset of the ads or access them programmatically.” However, we could not access the resource as of June 24th, 2019.
```
Comment (MattiSG) [inline _pages/en/google-transparency-report-assessment.md]: Could we add more details on how or why “we could not access the resource”? Did it yield a 404?

File: _pages/en/google-transparency-report-assessment.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+For instance, the ad [`CR517478000787193856`](https://transparencyreport.google.com/political-ads/advertiser/AR464795176020213760/creative/CR517478000787193856) was initially linked to advertiser `AR188817757448437760` (no longer accessible), then to advertiser `AR322201712037199872` (no longer accessible), and then to advertiser [`AR464795176020213760`](https://transparencyreport.google.com/political-ads/advertiser/AR464795176020213760). These 3 advertisers share the same name so we can assume that these 3 accounts are controlled by the same entity. However, the documentation does no explain why the identifier is not constant.
+
+### Ads content is not available programmatically
+
+Even though publishing the data as a bundle is a major advantage for research, the provided bundle only contains metadata about the ads and no content.
+
+Ads can be of three types: text, image or video.
+
+- We believe the textual content can be fetched using an undocumented Google API, which is cumbersome but feasible.
```
Comment (MattiSG) [inline _pages/en/google-transparency-report-assessment.md]: Can we elaborate?

File: _pages/en/google-transparency-report-assessment.md (hunk 1 above)
Comment (michelbl) [reply _pages/en/google-transparency-report-assessment.md]: See https://desinfo.quaidorsay.fr/ads/dumps/google/

File: _pages/en/google-transparency-report-assessment.md (hunk 2 above)
Comment (michelbl) [reply _pages/en/google-transparency-report-assessment.md]: Yields the error message "A required resource is not available."

File: _pages/en/google-transparency-report-assessment.md (hunk 3 above)
Comment (michelbl) [reply _pages/en/google-transparency-report-assessment.md]: On 1 example, I managed to retrieve the text, buried inside a deep and obscure data structure. I have no guarantee that the same method applies for all the text ads.

I agree that the current phasing is not very good. I think we can remove "we believe" without being intellectually dishonest.

**Comment by michelbl** [comment]
Hi @MattiSG . You can either merge as it is, make the changes, or wait until Wednesday for a second PR.

**Pull request merged**